src/openrct2/CmdlineSprite.cpp

```diff
@@ -46,9 +46,14 @@ assert_struct_size(rct_sprite_file_header, 8);
 
 #pragma pack(pop)
 
-static rct_sprite_file_header spriteFileHeader;
-static rct_g1_element* spriteFileEntries;
-static uint8_t* spriteFileData;
+struct SpriteFile
+{
+    rct_sprite_file_header Header;
```

Comment:
(On commit https://github.com/OpenRCT2/OpenRCT2/pull/13590/commits/ade064674ad5327edddbf9f5337eee72d7a5cdd6)

I can't comment on the `rct_sprite_file_header` declaration & definition, but it should have the member variables initialized to 0